README.md

```diff
@@ -1,182 +1,177 @@
--   ~~[Firestore//lastSupportedVersion](https://github.com/eclipse/thingweb.node-wot/blob/v0.8.6/packages/binding-firestore/README.md) :heavy_check_mark:~~
--   [Websocket](https://github.com/eclipse-thingweb/node-wot/tree/master/packages/binding-websockets) :heavy_plus_sign: (Server only)
--   [OPC-UA](https://github.com/eclipse-thingweb/node-wot/blob/master/packages/binding-opcua/README.md) :heavy_plus_sign: (Client only)
--   [NETCONF](https://github.com/eclipse-thingweb/node-wot/blob/master/packages/binding-netconf/README.md) :heavy_plus_sign: (Client only)
--   [Modbus](https://github.com/eclipse-thingweb/node-wot/tree/master/packages/binding-modbus) :heavy_plus_sign: (Client only)
--   [M-Bus](https://github.com/eclipse-thingweb/node-wot/tree/master/packages/binding-mbus) :heavy_plus_sign: (Client only)
+-   [Node.js](https://nodejs.org/) version 18+
+-   [npm](https://www.npmjs.com/) version 9+
 
-Note: More protocols can be easily added by implementing `ProtocolClient`, `ProtocolClientFactory`, and `ProtocolServer` interface.
+Platforms specific prerequisites:
 
-Note: The bindings for [binding-fujitsu](https://github.com/eclipse-thingweb/node-wot/tree/v0.7.x/packages/binding-fujitsu) and [binding-oracle](https://github.com/eclipse-thingweb/node-wot/tree/v0.7.x/packages/binding-oracle) were removed after `v0.7.x` due to lack of maintainers.
+-   Linux: Meet the [node-gyp](https://github.com/nodejs/node-gyp#installation) requirements:
+    -   Python v3.6, v3.7, or v3.8
+    -   make
+    -   A proper C/C++ compiler toolchain, like GCC
+-   Windows: Install the Windows build tools through a CMD shell as administrator:
+    -   `npm install -g --production windows-build-tools`
+-   Mac OS: Meet the [node-gyp](https://github.com/nodejs/node-gyp#installation) requirements:
+    -   `xcode-select --install`
 
-### MediaType Support
+If you want to use node-wot as a library in your Node.js application, you can use npm to install the node-wot packages that you need. To do so, `cd` inside your application folder, and run:
 
--   JSON :heavy_check_mark:
--   Text (HTML, CSS, XML, SVG) :heavy_check_mark:
--   Base64 (PNG, JPEG, GIF) :heavy_check_mark:
--   Octet stream :heavy_check_mark:
--   CBOR :heavy_check_mark:
--   EXI :timer_clock:
+```
+npm i @node-wot/core @node-wot/binding-http --save
+```
 
-Note: More mediaTypes can be easily added by implementing `ContentCodec` interface.
+#### Browser
 
-```JavaScript
-const ContentSerdes = require('@node-wot/core').ContentSerdes
-const JsonCodec = require('@node-wot/core').JsonCodec
+To use node-wot as a browser-side JavaScript Library, the browser needs to support ECMAScript 2015.
+Supported browsers include:
 
-// e.g., assign built-in codec for *new* contentType
-let cs = ContentSerdes.get();
-cs.addCodec(new JsonCodec("application/calendar+json"));
+-   Microsoft Edge 15 and later
+-   Firefox 54 and later
+-   Chrome 58 and later
+-   Safari 10 and later
```

Comment:
Do we actually need the list of browsers and version? I would stick to the first sentence "To use node-wot as a browser-side JavaScript Library, the browser needs to support ECMAScript 2015." only

Reply:
Considering that we didn't actually test node-wot in all these browsers and we don't have compliance tests I agree to remove it.

Reply:
I can't do it easly now, maybe in a followup quick PR?

Reply:
created https://github.com/eclipse-thingweb/node-wot/issues/1115